src/core/scene/a-scene.js

```diff
@@ -118,7 +118,12 @@ module.exports.AScene = registerElement('a-scene', {
         window.addEventListener('vrdisplaypresentchange', this.onVRPresentChangeBound);
 
         // Enter VR on `vrdisplayactivate` (e.g. putting on Rift headset).
-        window.addEventListener('vrdisplayactivate', function () { self.enterVR(); });
+        window.addEventListener('vrdisplayactivate', function () {
```

Comment:
This still seems to work.  Note that there is a quirk with VR-to-VR back navigation, but I think that is on the Oculus Browser side.
`works for me, although once you use back button to go back within VR, the next attempt to go forward again while staying in VR seems to fail, you get kicked back to 2D browser mode.  But if you then reload and re-enter VR, next in-VR traversal succeeds.`